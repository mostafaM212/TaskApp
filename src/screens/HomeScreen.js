import { StyleSheet, Text, View , Button } from 'react-native'
import React from 'react'

const HomeScreen = ({navigation}) => {
  return (
    <View style={styles.container}>
      <Button title='take photo' onPress={()=>navigation.navigate('CameraScreen')}/>
      <Text>HomeScreen</Text>
    </View>
  )
}

export default HomeScreen

const styles = StyleSheet.create({
  container : {
    justifyContent : 'center',
    alignItems: 'center',
    height : '100%'
  }
})