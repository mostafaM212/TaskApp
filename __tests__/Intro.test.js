import React from 'react';
import renderer from 'react-test-renderer';
import Intro from '../src/components/Intro';
import HomeScreen from '../src/screens/HomeScreen';

test('renders correctly', () => {
  const tree = renderer.create(<Intro />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('renders correctly', () => {
    
    const tree = renderer.create(<HomeScreen />).toJSON();

    expect(tree).toMatchSnapshot()
})