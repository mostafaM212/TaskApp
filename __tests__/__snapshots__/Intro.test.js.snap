// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    Welcome to React Native!
  </Text>
  <Text
    style={
      Object {
        "color": "#333333",
        "marginBottom": 5,
        "textAlign": "center",
      }
    }
  >
    This is a React Native snapshot test.
  </Text>
</View>
`;

exports[`renders correctly 2`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "height": "100%",
      "justifyContent": "center",
    }
  }
>
  <View
    accessibilityRole="button"
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        take photo
      </Text>
    </View>
  </View>
  <Text>
    HomeScreen
  </Text>
</View>
`;
